test/relations.test.js

```diff
@@ -1776,6 +1776,163 @@ describe('relations', function() {
     });
   });
 
+  describe('polymorphic relations validation', function() {
+    var validationError;
+    // define a mockup getRelationValidationMsg() method to log the validation error
+    var logRelationValidationError = function(code, rType, rName) {
+      validationError = {code, rType, rName};
+    };
+
+    it('rejects belongsTo relation if `model` is not provided', function() {
+      try {
+        var Picture = db.define('Picture', {name: String}, {relations: {
+          author: {
+            type: 'belongsTo',
+            foreignKey: 'authorId',
+            getRelationValidationMsg: logRelationValidationError},
+        }});
+      } catch (e) {
```

Comment:
@ebarault I think I find out how to catch the error, we might don't have to inject a mock function.

```js
      (function() {
        db.define('Picture', {name: String}, {relations: {
          author: {
            type: 'belongsTo',
            foreignKey: 'authorId',
          },
        }});
      }).should.throw('belongsTo relation: author requires param `model`');
```